### PR TITLE
Enhance current state of blacklist

### PR DESCRIPTION
**Pull request by jaivigneshvenugopal**
Sync works!

### CODE REVIEW

**Review by jelneo — APPROVED** [review]
Great job viggy! Keep it up.

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -118,8 +118,11 @@ private static Person createEditedPerson(ReadOnlyPerson personToEdit,
         Deadline updatedDeadline = editPersonDescriptor.getDeadline().orElse(personToEdit.getDeadline());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedPostalCode,
+        Person editedPerson = new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedPostalCode,
                 updatedDebt, updatedInterest, updatedDeadline, updatedTags);
+        editedPerson.setIsBlacklisted(personToEdit.getIsBlacklisted());
```
Comment (lawwman) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: just to clarify, this is just to make sure if the person was originally blacklisted, he would remain so after the edit?

**Review by lawwman — APPROVED** [review]
well done jaivignesh

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (jaivigneshvenugopal) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: Yes exactly!

**Review by khooroko — APPROVED** [review]
seems good! good job viggy

**Pull request merged**